automatic/flashplayerppapi/tools/ChocolateyInstall.ps1

```diff
@@ -1,9 +1,16 @@
-﻿$packageName = '{{PackageName}}'
-$version = '{{PackageVersion}}'
-$installerType = 'exe'
-$url = '{{DownloadUrl}}'
-$silentArgs = '-install'
-$validExitCodes = @(0)
+$packageName = 'flashplayerppapi'
+$version = '24.0.0.194'
+$majorVersion = '24'
```

Comment:
since $version and $majorVersion here isn't used anywhere, they can pretty much be removed.